### ISSUE TITLE
REGEXP_LIKE constraint not properly translated to PostgreSQL

### ISSUE THREAD

**Issue by mperdikeas** [darold/ora2pg#1316]
Using Ora2Pg v23.0
In migrating an Oracle database to PostgreSQL there's some serious mangling in REGEXP_LIKE check constraints.
In particular, the following Oracle SQL:

`CONSTRAINT "constraint_name" CHECK ( NOT (REGEXP_LIKE (some_column, '[,;!@#$%]') AND recordtype = 1)) DEFERRABLE INITIALLY DEFERRED ENABLE, `

... becomes:

`ALTER TABLE ... ADD CONSTRAINT constraint_name CHECK ( NOT (regexp_match(some_column, '[,'n') IS NOT NULL AND recordtype = 1)); `

The generated PostgreSQL statement is syntactically erroneous, e.g. psql complains with:

```
ERROR:  syntax error at or near "n"
LINE 1: ...constraint_name CHECK ( NOT (regexp_match(some_column, '[,'n') IS NOT...

```

**Comment by darold** [comment]
Right, this was a call of the wrong code transformation function. Fixed by commit 2fcaa2f.

**Comment by mperdikeas** [comment]
While we are at it, is it the same issue as the following ?

`CONSTRAINT` "..." CHECK (REGEXP_LIKE(column_name, '^(0[1-9]|[12][0-9]|3[01])[/](0[1-9]|1[012])[/](19|20)\d\d$')) ENABLE,

transforms to: 

`ALTER` TABLE ... ADD CONSTRAINT ... CHECK (regexp_match(column_name, '^(0[1-9]|[12][0-9]|3[01],'n') IS NOT NULL[/](0[1-9]|1[012])[/](19|20)\d\d$'));

> Right, this was a call of the wrong code transformation function. Fixed by commit [2fcaa2f](https://github.com/darold/ora2pg/commit/2fcaa2fd695d3a5b82601d48e2cc072cf7ad652f).

**Comment by darold** [comment]
Yes probably.
